lib/sprockets/server.rb

```diff
@@ -23,7 +23,7 @@ def call(env)
       start_time = Time.now.to_f
       time_elapsed = lambda { ((Time.now.to_f - start_time) * 1000).to_i }
 
-      if env['REQUEST_METHOD'] != 'GET'
+      if !['GET', 'HEAD'].include?(env['REQUEST_METHOD'])
```

Comment:
@ixti feel free to submit a pull request if you want. If you don't I'll do it myself.